### ISSUE TITLE
The example in readme does not compile

### ISSUE THREAD

**Issue by h8liu** [jdkato/prose#22]
I am referring to this:

```
package main
import "gopkg.in/jdkato/prose.v2"
func main() {
    doc, err := prose.NewDocument("Go is an open-source programming language created at Google.")
```

The `NewDocument` is actually in `gopkg.in/jdkato/prose.v2/summarize` now.

However, `go get gopkg.in/jdkato/prose.v2/summarize` does not work either. The package does not compile due to usage of internal package. This is a typical error when using the `gopkg.in` service. `gopkg.in` only redirects the git URI, but does not rewrite package import paths. As a result, the referred imports will actually point back to the `master` branch, nullifying the essential purpose of versioning. To use `gopkg.in` properly, you need to manually rewrite the import paths in the entire repo (or just stop using `gopkg.in` for multi package repos..).

I saw your repo from [Hacker News](https://news.ycombinator.com/item?id=17548768), but your repo fails to build on [smallrepo](https://smallrepo.com). Detailed build log here:

https://smallrepo.com/builds/20180717-175536-bc73d63d

Thanks.

**Comment by jdkato** [comment]
v2.0.0 isn't multi-package (and doesn't contain an internal package either). You can see the new source on the v2 branch, which gopkg.in points to.

**Comment by h8liu** [comment]
is this then a `gopkg.in` bug?

fwiw, the master branch is tracked on smallrepo [here](https://smallrepo.com/repos/github.com/jdkato/prose)

**Comment by h8liu** [comment]
@niemeyer, why ` gopkg.in/jdkato/prose.v2` is not pointing to the v2 branch?

**Comment by h8liu** [comment]
ah.. @jdkato  I think the reason is that you have a `v2.0.0` tag, which has higher priority over the v2 branch.

**Comment by jdkato** [comment]
You're right -- it appears that the `v2.0.0` tag was the culprit. Can you verify that this is indeed fixed?

**Comment by h8liu** [comment]
it fetches v2 now.

the `cmd/prose/main.go` still imports the non-v2 path: `github.com/jdkato/prose`
